app/views/posts/show.html.erb

```diff
@@ -1 +1,13 @@
-<h1>This is the page of the post with id: <%= @post_id %> of the user with id <%= @user_id %></h1>
\ No newline at end of file
+<section>
+    <h3>Post #<%= @post.id %> by <%= @user.Name %></h3>
+    <div>
+        <p>Comments: <%=@post.CommentsCounter %></p>
+        <p>Likes: <%=@post.LikesCounter %></p>
+    </div>
+    <p><%= @post.Text %></p>
+</section>
+<section>    
+    <% @post.recent_comment.each do |comment|%>
+       <p><%= comment.user.Name %>: <%= comment.Text %> </p>
+    <% end %>
+</section>
```

Comment:
- [ ] [REQUIRED] It will be nice if Post show page displays post with all comments listed. As you can see in this screen shoot ⏬ ⏬ , your post show page displays only 5 comments for the post id=2. In fact it should display 6 comments as post with id=2 has 6 comments.
![comments](https://user-images.githubusercontent.com/58552748/172311619-d436bd6b-7673-49f8-8061-31598f1a0bb4.png)